src/GraphQL.SystemTextJson/GraphQL.SystemTextJson.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
+    <RootNamespace>GraphQL.SystemTextJson</RootNamespace>
+    <Description>System.Text.Json serializer for GraphQL.NET</Description>
+    <PackageTags>GraphQL;json</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\..\assets\logo.64x64.png" Pack="true" PackagePath="\"/>
```

Comment:
Move to `Directory.Build.props`?

Reply:
I've just copied what NewtonsoftJson project did here.

Reply:
Ok